docs/eBpfExtensions.md

```diff
@@ -100,6 +100,39 @@ structure from provided data and context buffers.
 * `context_destroy`: Pointer to `ebpf_program_context_destroy_t` function that destroys a program type specific
 context structure and populates the returned data and context buffers.
 * `required_irql`: IRQL at which the eBPF program is invoked by bpf_prog_test_run_opts.
+* `capabilities`: 32-bit integer describing the optional capabilities / features supported by the extension.
+
+**Capabilities**
+
+`PROGRAM_DATA_CAPABILITY_CONTEXT_HEADER`:
+
+Flag indicating that extension supports adding a context header at the start of each context passed to the eBPF program.
```

Comment:
Maybe make it clear that the pointer passed to the BPF program is after the context header?

Reply:
Updated.